### DISCUSSION TITLE
p

### DISCUSSION THREAD

**Discussion by AD9881327** [e2nIEE/pandapower#1221] [Q&A]

**Comment by rbolgaryn** [comment]
Hi @AdityaDesai-09 ,

the N-1 Contingency can be analyzed in a loop, where you switch a line, run a load flow calculation, and switch the line on again, an repeating t for every line. 

As for the SC calculation, it is something you need to debug on your own, e.g. some of the parameters for the generators are missing, or a similar issue. You can look at an example in the tutorials (pandapower/tutorials in your installation) and see what parameters are necessary for SC calculation and compare to your grid model.

Roman
